requirements-dev.txt

```diff
@@ -1,9 +1,9 @@
-coverage==4.5.2
-flake8==3.6.0
+coverage==5.0.3
+flake8==3.7.9
 nose==1.3.7
 pep8==1.7.1
-pylint==1.9.2; python_version < '3.0'
```

Comment:
RIP python 2?

Reply:
I created a dedicated issue for that, Thank you